build-info-extractor/src/testFixtures/java/org/jfrog/build/IntegrationTestsBase.java

```diff
@@ -186,6 +182,9 @@ private boolean isRepoExists(String repo) throws IOException {
      * @throws IOException
      */
     private void createTestRepo(String repo) throws IOException {
+        if (StringUtils.isBlank(repo)) {
+            return;
+        }
         if (isRepoExists(repo)) {
             return;
         }
```

Comment:
The above two if statements can be merged into one.

Reply:
Merged.